pkg/shp/flags/build.go

```diff
@@ -14,14 +15,14 @@ func BuildSpecFromFlags(flags *pflag.FlagSet) *buildv1alpha1.BuildSpec {
 	spec := &buildv1alpha1.BuildSpec{
 		Source: buildv1alpha1.Source{
 			Credentials: &corev1.LocalObjectReference{},
-			Revision:    pointer.String(""),
-			ContextDir:  pointer.String(""),
+			Revision:    pointer.StringPtr(""),
```

Comment:
What's the reason for this change?

Reply:
I think there was an issue when one of the dependencies was updated.